packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -0,0 +1,38 @@
+/* Common Tag styles */
+.tag {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--size-tiny);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--size-tiny) var(--size-xs);
```

Comment:
Might be better to use `em` instead of `rem` to change in relation to changes in `font-size`.

In tag's case, using `padding: .5em 1em;` would most likely yield preferred results.

Reply:
Refactored to match the design and use em